instances/latest/terminologies/programmingLanguage/IGORPro.jsonld

```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/programmingLanguage/IGORPro",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/ProgrammingLanguage",
+  "definition": null,
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "IGOR Pro",
```

Comment:
keep "IGOR Pro:  as name we can "IGOR Pro programming language" to the synonym (@apdavison we decided that we did not raise the issue in other cases, e.g. MATLAB, R)